src/sentry/api/endpoints/organization_dashboards.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+
+from django.db import IntegrityError, transaction
+from sentry.api.base import DocSection
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.bases.dashboard import DashboardSerializer
+from sentry.api.serializers import serialize
+from sentry.models import Dashboard
+from rest_framework.response import Response
+
+
+class OrganizationDashboardsEndpoint(OrganizationEndpoint):
+    doc_section = DocSection.ORGANIZATIONS
+
+    def get(self, request, organization):
+        """
+        Retrieve an Organizations Dashboards
+        `````````````````````````````````````
+         Retrieve a list of dashboards that are associated with the given organization.
+         :pparam string organization_slug: the slug of the organization the
+                                          dashboards belongs to.
+        :qparam string query: the title of the dashboard being searched for.
+        :auth: required
+        """
+        dashboards = Dashboard.objects.filter(
+            organization_id=organization.id
+        )
+        query = request.GET.get('query')
+        if query:
+            dashboards = dashboards.objects.filter(
+                title=query,
```

Comment:
Do we want this to be an exact match? I imagine we won't have many dashboards in any given org, so we could do `startswith`, or even `contains` depending how fuzzy we want to be